sdk/formrecognizer/azure-ai-formrecognizer/src/samples/README.md

```diff
@@ -28,11 +28,16 @@ All of these samples need the endpoint to your Form Recognizer resource ([instru
 |**File Name**|**Description**|
 |----------------|-------------|
 |[Authentication][authentication_sample]|Authenticate the client|
-|[RecognizeContent][recognize_content] and [RecognizeContentFromUrlAsync][recognize_content_from_url_async]|Recognize text and table structures of a document|
+|[RecognizeBusinessCard][recognize_business_card] and [RecognizeBusinessCardAsync][recognize_business_card_async]|Recognize business card from a input stream|
+|[RecognizeBusinessCardFromUrl][recognize_business_card_from_url] and [RecognizeBusinessCardFromUrlAsync][recognize_business_card_from_url_async]|Recognize business card from an URL|
```

Comment:
nit, "a URL" is actually correct. 

```suggestion
|[RecognizeBusinessCardFromUrl][recognize_business_card_from_url] and [RecognizeBusinessCardFromUrlAsync][recognize_business_card_from_url_async]|Recognize business card from a URL|
```

Reply:
nooo that was my suggestion :( snif. Sorry @mssfang 
may I ask why? shouldn't `an` go before a word that starts with a vowel?

Reply:
This is the explanation I found:

>Although the U at the first of URL is a vowel, the abbreviation is pronounced “you-are-ell.” Because the letter Y is a consonant in this case, use a rather than an.

Reply:
I swear, english doesn't actually follow any of the rules it has 😆